### DISCUSSION TITLE
add options to allow email with SPF not pass to delivery to Inbox

### DISCUSSION THREAD

**Discussion by LagSeeN** [simple-login/app#844] [Feature Requests]
in some website not set SPF record or not update SPF record when i use aliase email not delivery to my Inbox because simplelogin can’t accept SPF fail and SPF none email to solve problem i need use real email to use that website

I love simplelogin but another service like forwardemail they accept email with SPF fail and SPF none delivery to my Inbox

**Comment by nguyenkims** [comment]
@LagSeeN whether to refuse an email that fails SPF is rather decided by the sender. So SimpleLogin only refuse emails if the sender wants emails from their domains that have SPF failed to be refused.

**Reply by LagSeeN** [reply]
thank and how [quarantine](https://app.simplelogin.io/dashboard/refused_email) it work can add refuse email with SPF not pass to quarantine ?

**Reply by nguyenkims** [reply]
This email is refused before by our spam scanner (SpamAssassin, rspamd) it reaches SimpleLogin email handler so we can't put it into quarantine. As we use a common spam scanner, this email is probably refused everywhere without ever making to the quarantine.
